### DISCUSSION TITLE
Allow configurable log levels

### DISCUSSION THREAD

**Discussion by justin8** [teslamate-org/teslamate#1971] [Ideas]
I've been using Teslamate for a while, but because I'm running it behind a reverse proxy I get several log lines per second from the healthchecks, making a lot of noise.

It would be great to be able to configure the log level to something other than `:info`, perhaps through an environment variable?

**Comment by djungelorm** [comment]
I'm also keen for this feature to be added.

I use a script that queries the index page from teslamate once per minute, to check it is operating correctly. But it generates a lot of noise in the logs with all the successful get requests.
